tokendito/helpers.py

```diff
@@ -416,40 +435,50 @@ def process_okta_aws_app_url(app_url=None):
     :param app_url: string with okta tile URL.
     :return: None.
     """
-    if app_url is None:
-        logging.error(
-            "Okta Application URL not found in profile '{}'.\nPlease verify your options"
-            " or re-run this application with the --configure flag".format(settings.okta_profile))
-        sys.exit(2)
-    # Prepare final Okta and AWS app Url
-    url = urlparse(app_url)
-
-    if url.scheme == '' or url.netloc == '' or url.path == '':
-        logging.error("Okta Application URL invalid. Please check your configuration"
-                      " and try again.")
+    if not validate_okta_aws_app_url(settings.okta_aws_app_url):
```

Comment:
either user app_url parameter here or remove the param to process_okta_aws_app_url entirely.

Reply:
fixed.